docs/cross-platform/macios/unified/updating-mac-apps.md

```diff
@@ -65,7 +65,7 @@ Add the following lines of code after the `<AssemblyName>` element:
 
 Example:
 
-![](updating-mac-apps-images/csproj3.png "Add these lines of code after the <AssemblyName> element")
+![](updating-mac-apps-images/csproj3.png "Add these lines of code after the \<AssemblyName> element")
```

Comment:
Escaping here is not required, the alt text renders correctly on the current published site:

![Screen Shot 2019-07-14 at 1 28 30 PM](https://user-images.githubusercontent.com/199373/61188956-7227c100-a63b-11e9-91d3-41936fc4b896.png)

Reply:
Ah, I found it was before in the alt text. This actually isn't in the alt text, but the title. Should it be moved to the alt text?

Reply:
I always get confused which maps to which in Markdown. No need to move it, I think we'll be doing an accessibility review soon to determine the best handling of title _and_ alt across the entire docset.

Reply:
Inside the bracket is alt text. The trailing string is title. Title doesn't work for Screen Readers or mobile.